projects/cashmere/src/lib/typeform-survey/typeform-survey.component.ts

```diff
@@ -57,7 +57,7 @@ export class TypeformSurveyComponent {
         if (!document.getElementById(this._id)) {
             this.getScripts();
         } else {
-            (<TypeformWindow>window).typeformEmbed.makePopup(this._fullUri, {
+            (<TypeformWindow>(<unknown>window)).typeformEmbed.makePopup(this._fullUri, {
```

Comment:
I know you are working with some existing code here, but ever since TypeScript introduced JSX support, the `<>` form of casting has been deprecated-ish.  I would rewrite it as `((window as unknown) as TypeformWindow).`

Reply:
hmm... looks like the TypeScript team has changed their position on this slightly: https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions
fwiw, it feels like we should try to be consistent as much as possible in Cashmere, and I would personally vote for the `as` syntax, but I won't hold up the review for it.